docs/guides/modules/permissions-authentication/pages/users-organizations-and-integrations-guide.adoc

```diff
@@ -0,0 +1,1092 @@
+= Users, organizations, and integrations guide
+:page-platform: Cloud
+:page-description: A guide to understanding user accounts, organizations and integrations with version control systems in CircleCI.
+:experimental:
+:page-aliases: guides:integration:github-apps-integration.adoc, guides:integration:github-integration.adoc, guides:integration:bitbucket-integration.adoc, guides:integration:gitlab-integration.adoc, guides:integration:bitbucket-data-center-integration.adoc
+
+This guide explains the concepts of user, organization, and integration in CircleCI. After these concepts are covered, you can find out about the features available to you depending on your organization and pipeline type.
+
+== User accounts
+
+When you log in to CircleCI you are logging in to your _user account_.
+
+As a CircleCI _user_ you can create and access _organizations_. You can access zero, one, or many organizations. You can create new organizations and/or join an existing organization.
+
+You can log in to CircleCI using one of the following methods:
+
+* Email and password
+* Social login (GitHub, Bitbucket)
+
+To view your user account integrations, navigate to menu:User Settings[Account Integrations].
+
+Select your icon in the top bar and select **User settings** to access the following:
+
+[#user-settings]
+[.table-scroll]
+--
+.User settings
+[cols="1,2", options="header"]
+|===
+| User settings section | What you will find here
+
+| Account Integrations
+| Displays your user ID as well as account integrations with VCS providers. You will find a list of which GitHub or Bitbucket organizations you are connected with. You can disconnect your OAuth connections with GitHub or Bitbucket here.
+
+| Notifications
+| Set your individual email and web notification preferences. This includes preferences around builds, branches, and project notifications. Web notifications will appear in your browser.
+
+| Accessibility
+| Toggle switch for accessible step output. Switch this on if you use a screen reader.
+
+| Privacy & Security
+| Disable third-party tracking. You may opt in or opt out of third party tracking pixels.
+
+| Sessions
+| A list of systems that have logged into your account. Revoke any sessions that you do not recognize.
+
+| Personal API tokens
+| View and create personal API tokens, used to access the CircleCI API.
+
+| Organization Plans
+| See the list of organizations you are a part of. If you have administrative privileges, you may also view the plan each organization is on.
+
+| Job SSH keys
+| Set up an SSH key to access your job containers. For more information on this feature see the xref:execution-managed:ssh-access-jobs.adoc[Debug with SSH] page.
+
+| Beta Program
+| Opt in to CircleCI's beta program. Beta features you opt in to will be listed on this page.
+
+|===
```

Comment:
For users that login with email/password there are two additional tabs, "Email" and "Password". 
<img width="281" height="503" alt="Screenshot 2025-11-25 at 13 15 52" src="https://github.com/user-attachments/assets/b6297287-80c9-4069-abec-e17eca118518" />